license_search.go

```diff
@@ -98,6 +98,10 @@ type LicenseSearchFuture struct {
 	m sync.Mutex
 }
 
+func (x *LicenseSearchFuture) LookupID() uint64 {
```

Comment:
This is inconsistent with the rest of the code. Look at `LicenseSearchResult` for example, it's got the `LookupID` field instead of a function.

Reply:
Fixed.